src/renderer/widgets/CreateWallet/ui/MultisigWallet/NameThresholdStep.tsx

```diff
@@ -0,0 +1,119 @@
+import { useUnit } from 'effector-react';
+import { useForm } from 'effector-forms';
+import { FormEvent } from 'react';
+
+import { Alert, Button, Input, InputHint, Select, SmallTitleText } from '@shared/ui';
+import { useI18n } from '@app/providers';
+import { DropdownOption } from '@shared/ui/types';
+import { networkModel, networkUtils } from '@entities/network';
+import { ChainTitle } from '@entities/chain';
+import { Signatory, type Chain } from '@shared/core';
+import { formModel } from '../../model/form-model';
+import { flowModel } from '../../model/flow-model';
+import { Step } from '../../lib/types';
+
+const getThresholdOptions = (optionsAmount: number): DropdownOption<number>[] => {
```

Comment:
Maybe move this two functions to utils?

Reply:
Do we use utils for UI components elsewhere? I've taken these 2 functions from the previous component. I can move them to some utils if we usually do this.